### BIG diffrences, this is a diff between CLUS "Constant Variable T" entry and topic selection in CLHS http://www.lispworks.com/documentation/lw70/CLHS/Body/a_t.htm#t
### +++ CLUS
### --- CLHS


--- saved-pages-dpans3/cl:constant_variables:t.txt	2017-04-27 21:57:10.000000000 +0100
+++ saved-pages-clus/cl:constant_variables:t.txt	2017-04-27 21:57:10.000000000 +0100
@@ -1,4 +1,85 @@
+Constant Variable T
+Constant Value
+t.
+Description
+The boolean representing true, and the canonical generalized boolean representing true. Although any object other than nil is considered true, t is generally used when there is no special reason to prefer one such object over another.
+The symbol t is also sometimes used for other purposes as well. For example, as the name of a class, as a designator (e.g. a stream designator) or as a special symbol for some syntactic reason (e.g. in case and typecase to label the otherwise-clause).
+Examples
+t 
+→
 T
-Please select which reference to T you intended
 
-Constant VariableSymbolSystem ClassGlossary Entry
\ No newline at end of file
+(eq t 't) 
+→
+true 
+(find-class 't) 
+→
+#<CLASS T 610703333> 
+(case 'a (a 1) (t 2)) 
+→
+1 
+(case 'b (a 1) (t 2)) 
+→
+2 
+(prin1 'hello t)
+▷
+HELLO 
+→
+HELLO 
+See Also
+ nil
+Notes
+None.
\ No newline at end of file
